gradle/instrumentation.gradle

```diff
@@ -9,14 +10,6 @@ ext {
   mavenGroupId = 'io.opentelemetry.javaagent.instrumentation'
 }
 
-byteBuddy {
-  transformation {
-    // Applying NoOp optimizes build by applying bytebuddy plugin to only compileJava task
-    tasks = ['compileJava', 'compileScala', 'compileKotlin']
-    plugin = 'io.opentelemetry.javaagent.tooling.muzzle.collector.MuzzleCodeGenerationPlugin$NoOp'
```

Comment:
Can we remove this `NoOp` class if it's no longer used?

Reply:
Removed it.